TradeItIosTicketSDK2/TradeItYahooCryptoTradingTicketViewController.swift

```diff
@@ -445,15 +445,18 @@ class TradeItYahooCryptoTradingTicketViewController:
                         cell?.configureQuantityType(
                             quantitySymbol: quantitySymbol,
                             quantity: self.order.quantity,
-                            maxDecimalPlaces: instrumentOrderCapabilities.maxDecimalPlacesFor(orderQuantityType: self.order.quantityType)
+                            maxDecimalPlaces: instrumentOrderCapabilities.maxDecimalPlacesFor(orderQuantityType: self.order.quantityType),
+                            showToggle: supportedOrderQuantityTypes.count > 1
                         )
                     }
                 }
             )
+            let supportedOrderQuantityTypes = instrumentOrderCapabilities.supportedOrderQuantityTypesFor(action: self.order.action)
             cell?.configureQuantityType(
                 quantitySymbol: quantitySymbol,
                 quantity: self.order.quantity,
```

Comment:
do we need to set this to `nil` as we do in the Equity controller?
https://github.com/tradingticket/TradeItIosTicketSDK2/blob/5dcc0ae3467ce2f127f7135ba7897a3a67897eca/TradeItIosTicketSDK2/TradeItYahooEquityTradingTicketViewController.swift#L437